toolchain/check/import_ref.cpp

```diff
@@ -73,6 +74,11 @@ class ImportRefResolver {
   }
 
  private:
+  struct Work {
+    SemIR::InstId inst_id;
+    bool incomplete_type = false;
```

Comment:
From reading the rest of the PR, I think I have some idea of how all this is used, but I think it's a somewhat hard to piece together. Could you add some comments, maybe on the `Work` type rather than just this field that try to pull together how this is used throughout the resolution?

Reply:
Added some comments here, moved the comments on TryResolveInst up to the class level and elaborated on them.